cpu/stm32_common/periph/uart.c

```diff
@@ -292,6 +329,19 @@ void uart_poweroff(uart_t uart)
 #endif
 }
 
+static uint8_t _data_mask(uart_t uart)
```

Comment:
I am still strongly in favor of spending one byte RAM per UART device for pre-calculating this mask instead of having these ~10 instructions overhead in each ISR call!

Reply:
Yup fair enough.